utils/build-mlir-aie-pcie.sh

```diff
@@ -53,7 +53,7 @@ if [ -z "$LIBXAIE_DIR" ] || [ -z "$HSAKMT_DIR" ]; then
   mkdir aie-rt-x86_64-hsa
   mkdir aie-rt-x86_64-hsa/lib/
   cd aie-rt
-  git checkout phoenix_v2023.2
+  git checkout strix_2024.2 
```

Comment:
The issue that I have always ran into is when we don't provide a `LibXAIE_${target}_DIR` it will pull in certain parts of LibXAIE from the Vitis install: https://github.com/Xilinx/mlir-aie/blob/main/runtime_lib/CMakeLists.txt#L48-L62 which will then fail as it doesn't recognize our custom aie-rt backend. I don't think we can just use the submodule without more CMake changes. If you think it is cleaner to just build `third_party/aie-rt` and pass that in as a LibXAIE target I am happy to do that.

Reply:
Nice yeah that is a good idea. I am just copying it into build and building it, I want to keep all of the build artifacts within the build directory. Addressed in [80721e7](https://github.com/Xilinx/mlir-aie/pull/1892/commits/80721e7da97bc2c874b7cc2b90b285c351a55449).